lms/djangoapps/bulk_email/tasks.py

```diff
@@ -652,27 +614,17 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
             to_list.pop()
 
         log.info(
-            "BulkEmail ==> Task: %s, SubTask: %s, EmailId: %s, Total Successful Recipients: %s/%s, \
-            Failed Recipients: %s/%s, Time Taken: %s",
-            parent_task_id,
-            task_id,
-            email_id,
-            total_recipients_successful,
-            total_recipients,
-            total_recipients_failed,
-            total_recipients,
-            time.time() - start_time
+            f"BulkEmail ==> Task: {parent_task_id}, SubTask: {task_id}, EmailId: {email_id}, Total Successful "
+            f"Recipients: {total_recipients_successful}/{total_recipients}, Failed Recipients: "
+            f"{total_recipients_failed}/{total_recipients}, Time Taken: {time.time() - start_time}"
         )
+
         duplicate_recipients = [f"{email} ({repetition})"
                                 for email, repetition in recipients_info.most_common() if repetition > 1]
         if duplicate_recipients:
             log.info(
-                "BulkEmail ==> Task: %s, SubTask: %s, EmailId: %s, Total Duplicate Recipients [%s]: [%s]",
-                parent_task_id,
-                task_id,
-                email_id,
-                len(duplicate_recipients),
-                ', '.join(duplicate_recipients)
+                f"BulkEmail ==> Task: {parent_task_id}, SubTask: {task_id}, EmailId: {email_id}, Total Duplicate "
+                f"Recipients [{len(duplicate_recipients)}]"
```

Comment:
Is it okay that we're losing the ability to see who the duplicate recipients are? Should we log user IDs?

Reply:
I thought it would be okay. I definitely didn't want to print the list of emails any longer. If we have to ever look into an issue with duplicate recipients we should have the email id (which includes the "targets" (recipient groups) for the message and the course information. I don't think we're losing too much info here removing it.

I thought about adding a debug log or conditional to print the list but decided to skip over that as well. 

I had checked in Splunk as well and couldn't find any evidence of duplicate recipients over the last 30 days in BulkEmail logs.

I'll think about it a bit more this morning but I don't think I am going to make any other updates at this time unless you strongly think we should.

Reply:
I think it's fine to leave as is. It was just a change in data that I wanted to call out.